shell/platform/tizen/flutter_tizen.cc

```diff
@@ -216,29 +164,40 @@ void FlutterRegisterViewFactory(
           view_type, std::move(view_factory)));
 }
 
+// Returns the texture registrar corresponding to the given opaque API handle.
+static FlutterTizenTextureRegistrar* TextureRegistrarFromHandle(
+    FlutterDesktopTextureRegistrarRef ref) {
+  return reinterpret_cast<FlutterTizenTextureRegistrar*>(ref);
+}
+
+// Returns the opaque API handle for the given texture registrar instance.
+static FlutterDesktopTextureRegistrarRef HandleForTextureRegistrar(
+    FlutterTizenTextureRegistrar* registrar) {
+  return reinterpret_cast<FlutterDesktopTextureRegistrarRef>(registrar);
+}
+
 FlutterDesktopTextureRegistrarRef FlutterDesktopRegistrarGetTextureRegistrar(
     FlutterDesktopPluginRegistrarRef registrar) {
-  FT_UNIMPLEMENTED();
-  return nullptr;
+  return HandleForTextureRegistrar(registrar->engine->GetTextureRegistrar());
```

Comment:
```texture_registrar_``` is initialized only when ```engine``` is headed, Don't we need to warn a case that ```engine``` is headless?

Reply:
Add a warning log in headless mode.

Reply:
As far as I remember FlutterDesktopRegistrarGetTextureRegistrar is called whenever an app is launched. Then does the change mean the warning is always shown when starting a headleas app?

Reply:
Is it because of the below implementation? 
https://github.com/flutter-tizen/engine/blob/9f408ef240340dc9ce271a6cfe84d68e04cf977f/shell/platform/common/cpp/client_wrapper/plugin_registrar.cc#L25

Should we do one of three things?
1. Always initialize ``texture_registrar_```
2. Warn a this case
3. Leave it as it is.

Reply:
@xiaowei-guan Could you revert the change related to this? A warning shouldn't be displayed when launching a headless app.

Reply:
Ok, I have removed this warning log

Reply:
Thank you!